server/package.json

```diff
@@ -31,7 +31,7 @@
     "sqlite3": "^5.1.6",
     "uuid": "^9.0.0",
     "jsonwebtoken": "^8.5.1",
-    "vectordb": "0.1.5-beta"
+    "vectordb": "0.1.5"
```

Comment:
Ah thanks for this, Lance team didnt tell me they made the beta changes available in 0.1.5 (x86 darwin was missing so they made a beta branch for us)

Reply:
I was getting some errors or warnings on the beta branch and they went away on 0.1.5. Did you test that package running node locally? I wasn't able to test on a Mac.

Reply:
I haven't but when I checked it out I saw the missing file that was breaking the v0.1.4 release so I can manually check tonight.